common/src/main/scala/org/apache/celeborn/common/RssConf.scala

```diff
@@ -1158,14 +1231,6 @@ object RssConf extends Logging {
     conf.getDouble("rss.columnar.shuffle.max.dict.factor", 0.3)
   }
 
-  // If we want to use multi-raft group we can
-  // add "rss.ha.service.ids" each for one raft group
-  val HA_SERVICE_ID_KEY = "rss.ha.service.id"
-  val HA_NODES_KEY = "rss.ha.nodes"
-  val HA_ADDRESS_KEY = "rss.ha.address"
-  val HA_RATIS_PORT_KEY = "rss.ha.port"
-  val HA_RATIS_PORT_DEFAULT = 9872
```

Comment:
let me clarify agian

- `rss.ha.service.id` is removed (functionality is dropped as well)
- `rss.ha.nodes` is removed (replaced w/ extraction mechanism)
- `rss.ha.address` is replaced w/ `celeborn.ha.master.node.<id>.ratis.host`
- `rss.ha.port` is replaced w/ `celeborn.ha.master.node.<id>.ratis.port`